### DISCUSSION TITLE
dolphinscheduler打通seatunnel-web建议

### DISCUSSION THREAD

**Discussion by zq0757** [apache/dolphinscheduler#16099] [Ideas]
设计上是在ds上加一个st的任务，可以调用st-web里配置的作业列表，然后运行的时候调用st-web的接口获取一个jobconfig文件，再通过seatunnel.sh提交这个文件

**Comment by soulgoast** [comment]
目前dolphinscheduler都是通过脚本来调用其他服务，这样就可以保证版本的兼容。如果改成接口，如何升级呢？
